README.md

```diff
@@ -95,15 +100,17 @@ output "aws_parameters" {
 
 
 | Name                           | Description                                                            | Type     | Default             | Required |
-| -------------------------------- | ------------------------------------------------------------------------ | ---------- | --------------------- | ---------- |
+| ------------------------------- | ------------------------------------------------------------------------ | ---------- | --------------------- | ---------- |
 | integration_name               | Prefix to be used for naming new resources                             | `string` | `UptycsIntegration` |          |
 | upt_account_id                 | Uptycs AWS account ID                                                  | `string` | `""`                | Yes      |
 | aws_account_id                 | AWS organization's master account ID                                   | `string` | `""`                | Yes      |
 | external_id                    | External ID                                                            | `uuid4`  | `""`                | Yes      |
 | vpc_flowlogs_bucket_name       | Name of the S3 bucket in master for VPC flow logs                      | `string` | `""`                | Optional |
-| cloudtrail_s3_bucket_name      | Name of the organization cloud trail S3 bucket                         | `string` | `""`                | Optional |
+| cloudtrail_s3_bucket_name      | Name of the organization's cloud trail S3 bucket                       | `string` | `""`                | Optional |
 | cloudtrail_s3_bucket_in_master | Specifies whether the cloudtrail s3 bucket is in master account or not | `bool`   | `true`              |          |
-| kinesis_stream_name            | Name of the organization Kinesis stream                                | `string` | `""`                | Optional |
+| cloudtrail_s3_bucket_account   | Child Account id in which the cloudtrail S3 bucket exists              | `string` | `""`                | Optional |
```

Comment:
```suggestion
| cloudtrail_s3_bucket_account   | Child Account id in which the cloudtrail S3 bucket exists ()              | `string` | `""`                | Required when cloudtrail_s3_bucket_in_master = false |
```

Reply:
Can you make sure we fail the case when cloudtrail_s3_bucket_in_master=false and this is not set?
Then you don't need to chane "Required column"